src/Components/cards/Cards.css

```diff
@@ -0,0 +1,115 @@
+:root {
+  --background-clr: #e5e7eb;
+  --clr-1: rgb(154, 0, 54);
+  --clr-2: rgb(17, 82, 147);
+}
+
+.container-flex {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 1rem;
+  margin-right: 0rem;
+  flex-direction: column;
+  align-content: center;
+  background-color: var(--background-clr);
+  padding: 0;
+}
+.cards-container {
+  width: 30%;
+  height: 30%;
+  align-self: center;
+  background-color: white;
+  margin-top: 2rem;
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+  border-bottom-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+  box-shadow: 1px 3px 12px 0px gray;
+}
+
+.heading {
+  text-align: center;
+}
+.card-flex-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  flex-direction: column;
+  border-top: lightgray;
+}
+.card-item {
+  margin: 0.5rem;
+  align-self: center;
+}
+
+.card-item-font {

```

Comment:
card-item-font can be made generalized so that it can be used anywhere we want large and bold text, I would suggest the name to be large-bold-text-style kind of something.